tests/unit/multiclass/LaRank_unittest.cc

```diff
@@ -6,56 +6,54 @@
 
 using namespace shogun;
 
-TEST(DISABLED_LaRank,train)
+TEST(LaRank,train)
 {
-	index_t num_vec=10;
-	index_t num_feat=3;
-	index_t num_class=num_feat; // to make data easy
-	float64_t distance=15;
+	index_t num_vec = 10;
```

Comment:
the best thing is to do a first patch with the bugfix, and a second with the whitespace changes